### PR TITLE
(for benchmarking) Lean pr testing 5920

### PR DESCRIPTION
**Pull request by kmill**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by kmill** [comment]
!bench

File: Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Order.lean [hunk 1]
```diff
@@ -267,9 +267,10 @@ lemma le_iff_norm_sqrt_mul_rpow {a b : A} (hbu : IsUnit b) (ha : 0 ≤ a) (hb :
   lift b to Aˣ using hbu
   have hbab : 0 ≤ (b : A) ^ (-(1 / 2) : ℝ) * a * (b : A) ^ (-(1 / 2) : ℝ) :=
     conjugate_nonneg_of_nonneg ha rpow_nonneg
+  #adaptation_note /-- 2024-11-10 added `(R := A)` -/
   conv_rhs =>
     rw [← sq_le_one_iff (norm_nonneg _), sq, ← CStarRing.norm_star_mul_self, star_mul,
-      IsSelfAdjoint.of_nonneg sqrt_nonneg, IsSelfAdjoint.of_nonneg rpow_nonneg,
+      IsSelfAdjoint.of_nonneg (R := A) sqrt_nonneg, IsSelfAdjoint.of_nonneg rpow_nonneg,
```
Comment (Vierkantor) [inline Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Order.lean:273]: This one is okay IMO: I wouldn't expect `0 ≤ ↑b ^ (-(1 / 2))` to unify with `0 ≤ sqrt _` in any circumstance anyway.

File: Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean [hunk 2]
```diff
@@ -174,14 +174,14 @@ theorem ruzsa_triangle_inequality_mul_mul_mul (A B C : Finset G) :
...
+  rw [← le_div_iff₀ (cast_pos.2 hB.card_pos), mul_div_right_comm, mul_comm _ B]
   refine (Nat.cast_le.2 <| card_le_card_mul_left _ hU.1).trans ?_
   refine le_trans ?_
     (mul_le_mul (hUA _ hB') (cast_le.2 <| card_le_card <| mul_subset_mul_right hU.2)
       (zero_le _) (zero_le _))
-  rw [← mul_div_right_comm, ← mul_assoc]
-  refine (le_div_iff₀ <| cast_pos.2 hU.1.card_pos).2 ?_
+  #adaptation_note /-- 2024-11-01 `le_div_iff₀` is synthesizing wrong `GroupWithZero` without `@` -/
+  rw [← mul_div_right_comm, ← mul_assoc,
+    @le_div_iff₀ _ (_) _ _ _ _ _ _ _ (cast_pos.2 hU.1.card_pos)]
```
[the remark this answers is not in the document]
Comment (Vierkantor) [reply Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean:184]: #18860

**Review by Vierkantor — APPROVED** [review]
If we either merge #18860 or we add it into this PR, the adaptations for leanprover/lean4#5920 look good to me.

**Pull request merged**